src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.UnitTests/Controllers/ProviderControllerTests.cs

```diff
@@ -0,0 +1,80 @@
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Api.Controllers;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetProvider;
+
+namespace SFA.DAS.CommitmentsV2.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    [Parallelizable]
+    public class ProviderControllerTests
```

Comment:
There is a danger here of failing tests, due to the Parallelizable attribute

see this article
nunit/nunit#2105

Options are to remove the Parallelizable attribute or instantiate the fixture per test. Both options are good.

Reply:
After discussion with Ian and Paul, this is not an issue. The parallelizable attribute on the class only makes the class run in parallel with other parallelizable classes. It doesn't make the elements within class run in parallel with each other, so this is not an issue.